sklearn/metrics/scorer.py

```diff
@@ -26,8 +26,9 @@
 from . import (r2_score, median_absolute_error, mean_absolute_error,
                mean_squared_error, mean_squared_log_error, accuracy_score,
                f1_score, roc_auc_score, average_precision_score,
-               precision_score, recall_score, log_loss, balanced_accuracy_score,
-               explained_variance_score, brier_score_loss)
+               precision_score, recall_score, log_loss,
```

Comment:
You probably added this change because of pep8, right? I'm just curious why this change wasn't added in a previous PR.

Reply:
Not sure why it landed up here...

Reply:
I'm forced by Travis o(╥﹏╥)o Let me try to remove it.